code/modules/jobs/job_types/bartender.dm

```diff
@@ -18,6 +18,13 @@
 
 	family_heirlooms = list(/obj/item/reagent_containers/glass/rag, /obj/item/clothing/head/that, /obj/item/reagent_containers/food/drinks/shaker)
 
+	mail_goodies = list(
+		/obj/item/storage/box/rubbershot = 30,
+		/datum/reagent/consumable/clownstears = 10,
```

Comment:
Instead of special casing reagents, just make a new subtype of beakers containing the reagent. Then you can add bemusing descriptions.